charts/rancher-monitoring/v0.0.7000/charts/webhook-receiver/templates/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.version" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "app.fullname" . }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "app.name" . }}
+      chart: {{ template "app.version" . }}
```

Comment:
I would not recommended to add `chart: {{ template "app.version" . }}` as a match label in both `.spec.selector.matchLabels` and `.spec.template.metadata.labels`, because bump the chart version will result in an immutable fields error, however, the app version may never get updated since this is a legacy design issue of the monitoring chart itself.